website/www/site/content/en/documentation/io/connectors.md

```diff
@@ -559,7 +559,13 @@ This table provides a consolidated, at-a-glance overview of the available built-
       ✔
       <a href="https://beam.apache.org/releases/pydoc/current/apache_beam.io.gcp.bigtableio.html">via X-language</a>
     </td>
-    <td>Not available</td>
+    <td class="present">
+      ✔
+      <a href="https://pkg.go.dev/github.com/apache/beam/sdks/v2@v2.51.0/go/pkg/beam/io/bigtableio">native</a> (sink)
+      <br>
+      ✔
+      <a href="https://pkg.go.dev/github.com/apache/beam/sdks/v2@v2.51.0/go/pkg/beam/io/xlang/bigtableio">via X-language</a>
```

Comment:
Remove the specific version (the @2.51.0) so they will link to the latest release versions instead.